cfgov/unprocessed/css/organisms/header.less

```diff
@@ -62,7 +61,11 @@
 */
 
 .o-header {
+    min-width: 320px;
```

Comment:
Probably more atomic to set it on the element that needs it. I could be convinced otherwise though, but the argument would be if you take the header organism on its own it'll break if it gets below 320px (logo will get cropped), so the organism taken in isolation has a min width. 

Having a variable for this seems like a good idea.